addons/scratchr2/scratchr2_comments.css

```
@@ -180,3 +180,32 @@
   /* emoji-picker compatibility */
   width: auto;
 }
+#comments .control-group.tooltip {
+  display: flex;
+  flex-direction: column;
+}
+#comments .control-group.tooltip textarea {
+  margin-bottom: 0.5rem;
```

```suggestion
  margin-bottom: 8px;
```

`1rem` on scratchr2 pages isn't the same size as in scratch-www.